py/torch_tensorrt/dynamo/_exporter.py

```diff
@@ -411,3 +420,111 @@ def inline_trt_modules(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
         gm.graph.erase_node(trt_module_node)
 
     return gm
+
+
+def inline_trt_modules_for_windows(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
+    """
+    Replace TRT submodules with trt engine nodes.
+    """
+    for name, _ in gm.named_children():
+        if "_run_on_acc" not in name:
+            continue
+        # Get the TRT submodule
+        trt_module = getattr(gm, name)
+
+        # Ensure the trt module node in the main graph (gm) has inputs
+        trt_module_node = [node for node in gm.graph.nodes if node.name == name]
+        assert trt_module_node
+        trt_module_node = trt_module_node[0]
+        assert trt_module_node.args
+
+        if "val" not in trt_module_node.meta:
+            raise ValueError(
+                f"trt_module_node: {trt_module_node.name} does not have the metadata which should be set during dynamo compile_module step."
+            )
+        num_outputs = len(trt_module_node.meta["val"])
+        # Insert a call_function node to perform inference on TRT engine
+        with gm.graph.inserting_before(trt_module_node):
+            engine_info = trt_module._pack_engine_info()
+            engine_bytes = engine_info[ENGINE_IDX]
+            engine_info[ENGINE_IDX] = base64.b64encode(engine_bytes).decode("utf-8")
+            # insert the no_placeholder node in the graph which should be replaced to the actual execute_engine node while load in the windows
+            trt_node = gm.graph.call_function(
+                torch.ops.tensorrt.no_op_placeholder_for_execute_engine.default,
+                (trt_module_node.args, *engine_info),
+            )
```

Comment:
I think this is the only difference b/w inline_trt_modules_for_windows and inline_trt_modules. So merge these two functions and gate them via cross_compile flag